File: 프로그래머스/0/181841. 꼬리 문자열/꼬리 문자열.cs
using System;

public class Solution 
{
    public string solution(string[] str_list, string ex) 
    {
        string connectStr = "";
        
        for(int i=0; i<str_list.Length; i++)
        {
            if(str_list[i].Contains(ex))
            {
                continue;
            }
            connectStr += str_list[i];
        }
        return connectStr;
    }
}

File: 프로그래머스/0/181841. 꼬리 문자열/꼬리 문자열.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

string solution(vector<string> str_list, string ex) 
{
    string result = "";
    for(int i=0; i<str_list.size(); i++)
    {
        string str = str_list[i];
        if(search(str.begin(), str.end(), ex.begin(), ex.end()) != str.end())
        {
            continue;
        }
        result += str;
    }
    return result;
}